src/AspNet.Security.OAuth.Discord/DiscordAuthenticationOptions.cs

```diff
@@ -26,9 +27,12 @@ public DiscordAuthenticationOptions()
             TokenEndpoint = DiscordAuthenticationDefaults.TokenEndpoint;
             UserInformationEndpoint = DiscordAuthenticationDefaults.UserInformationEndpoint;
 
-            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "id");
-            ClaimActions.MapJsonKey(ClaimTypes.Name, "username");
-            ClaimActions.MapJsonKey(ClaimTypes.Email, "email");
+			ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "id");
+			ClaimActions.MapJsonKey(ClaimTypes.Name, "username");
+			ClaimActions.MapJsonKey(ClaimTypes.Email, "email");
+			ClaimActions.MapJsonKey(Claims.AvatarHash, "avatar");
+			ClaimActions.MapCustomJson(Claims.AvatarUrl, user => $"{Urls.DiscordCDN}/avatars/{user.GetString("id")}/{user.GetString("avatar")}.png");
```

Comment:
Add a property for the CDN host/URL to the options, assign it the value of the constant by default, and then consume it from there.

That way, if Discord were to change it for some reason, it can be easily changed by the consumer, rather then it being effectively hard-coded here.

Reply:
I've completed the change as requested, and added a DiscordCDN property to the DiscordAuthenticationOptions. I've also included a property for overriding the URL format of the avatar. Please let me know if this is fine.